pyglet/gui/widgets.py

```diff
@@ -445,7 +445,7 @@ def __init__(self, text: str,
                 Optional parent group of text entry widget.
         """
         self._doc = pyglet.text.document.UnformattedDocument(text)
-        self._doc.set_style(0, len(self._doc.text), dict(color=text_color))
```

Comment:
Doesn't the linter prefer `dict(...)` instead of `{...}`?

Reply:
it prefer `{...}` better